azure_provisioning_e2e/iothubservice20180630/service_helper.py

```diff
@@ -0,0 +1,173 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
I recommend not mixing autogen  code with hand-written code in the same directory.

Reply:
+1

---
In reply to: [314840267](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r314840267) [](ancestors = 314840267)